core/common/src/main/java/alluxio/wire/MasterInfo.java

```diff
@@ -0,0 +1,110 @@
+/*
+ * The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+ * (the "License"). You may not use this work except in compliance with the License, which is
+ * available at www.apache.org/licenses/LICENSE-2.0
+ *
+ * This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied, as more fully set forth in the License.
+ *
+ * See the NOTICE file distributed with this work for information regarding copyright ownership.
+ */
+
+package alluxio.wire;
+
+import com.google.common.base.Objects;
+
+import java.io.Serializable;
+
+import javax.annotation.concurrent.NotThreadSafe;
+
+/**
+ * Information about the Alluxio master.
+ */
+@NotThreadSafe
+public final class MasterInfo implements Serializable {
+  private static final long serialVersionUID = 5846173765139223974L;
```

Comment:
Eclipse generates them. It doesn't matter what they are, only that we don't change them. Setting them all to 1 would also be fine, but our convention is the big numbers for some reason